tests/commands/test_commands.py

```diff
@@ -280,8 +277,7 @@ def test_calling_main(start_mock, base_parser_mock, parse_args_mock,
 @patch('bigchaindb.config_utils.autoconfigure')
 @patch('bigchaindb.commands.bigchaindb.run_recover')
 @patch('bigchaindb.start.start')
-def test_recover_db_on_start(mock_autoconfigure,
```

Comment:
why was `mock_autoconfigure` removed?

Reply:
oh, that might have been a mistake. I see that the patch is still in. I need to check. If I removed it then because it was not needed, but I need to check again.

Reply:
ok, so I can confirm now, that it is not needed. I'll remove the patch in the next commit